pymodm/base/fields.py

```diff
@@ -267,6 +267,37 @@ def _model_to_document(self, value):
             '%s is not a valid %s' % (value, self.related_model.__name__))
 
 
+class RelatedEmbeddedModelFieldsBase(RelatedModelFieldsBase):
+    '''Base class for EmbeddedDocument and EmbeddedDocumentListField.'''
```

Comment:
Can you change to double quotes (`"""..."""`)?

Reply:
@ShaneHarvey done this.